src/main.rs

```diff
@@ -1,15 +1,89 @@
+use gloo_render::AnimationFrame;
 use nes_emu::Nes;
-use yew::prelude::*;
+use std::pin::Pin;
+use wasm_bindgen::JsCast;
```

Comment:
what is the need for this import ? It doesn't seem to be used

Reply:
It's a trait, so it enables a cast method that's used with NodeRef I think, not sure, but clippy didn't complain

Reply:
Ok. Right. I forgot about the traits. Nothing to complaint then. It Can be merged as soon as you wish